design/protocol-changes/ledger.tex

```diff
@@ -23,16 +27,17 @@ \subsection{Analysis}
 This means that, on average (assuming no cooldown periods), an attacker with sufficient stake to be elected at least once per round on average\footnote{Concretely, this is satisfied for an adversary with stake $\alpha$ if $\phi(\alpha)\cdot\perasRoundSlots \ge 1$, so for $\alpha \ge 21.8\%$ if $f=1/20$ and $\perasRoundSlots=90$.} can include
 \[\frac \perasCertSizeLimit \perasRoundSlots\,\unit{B/\slot}\]
 without anyone paying for the associated cost of bandwidth/storage.
-However,
 
 For comparison, the fee for including $\perasCertSizeLimit$ bytes on the chain as part of a transaction is given by $\mi{minfeeA}\cdot \perasCertSizeLimit$.
-For realistic values on the conservative end (using $\mi{minfeeA} = 44$ and $\mi{minfeeB} = 155381$ on Cardano mainnet as of epoch 537\footnote{See e.g.\ \url{https://cexplorer.io/params}.}, $\perasRoundSlots=90$ and $\perasCertSizeLimit = \SI{50}{kB}$), this would correspond to a cost of
-\[ \frac{\mi{minfeeA\cdot \perasCertSizeLimit + \mi{minfeeB}}}{\perasRoundSlots} = \SI{26170.9}{\lovelace/\slot} \approx \SI{94.22}{\ADA/h} \]
+For realistic values on the conservative end (using $\mi{minfeeA} = 44$ and $\mi{minfeeB} = 155381$ on Cardano mainnet as of epoch 537\footnote{See e.g.\ \url{https://cexplorer.io/params}.}, $\perasRoundSlots=90$ and $\perasCertSizeLimit = \SI{20}{kB}$, cf.~\cref{fig:vote cert metrics}), this would correspond to a cost of
```

Comment:
I see we have both `\autoref` and `\cref` in the document; shall we pick one?

Reply:
I slightly prefer cleveref as it is [more powerful](https://tex.stackexchange.com/a/119222), but I guess either works for our purposes.

Reply:
Addressed in #34